### ISSUE TITLE
Chemicals - Gas Fumes for mixing

### ISSUE THREAD

**Issue by Rushi123456** [civilnetworks-projects/SCP_Public_Development#453]
https://www.civilgamers.com/community/threads/gas-fumes-from-chemical-mixing.17184/

**Description:** Add an ability to crafting menu so that it adds an ability for it to create fumes on a recipe, that are deadly. Using gas masks would counter this gas fumes.

**Comment by GarrySwepman** [comment]
I don't think this counts as a bug, but hey cool suggestion would like to see it in-game!

**Comment by McYeke** [comment]
Closed not planned
